src/env/index.ts

```diff
@@ -27,6 +27,8 @@ export const REGION_JSON_URL = Config.REGION_JSON_URL;
 
 export const EN_CONFIG_URL = Config.EN_CONFIG_URL;
 
+export const EN_API_VERSION = Config.EN_API_VERSION;
```

Comment:
`.env.example` should include an example of EN_API_VERSION with acceptable values.

Reply:
Good call!